README.md

```diff
@@ -122,28 +129,28 @@ The interaction with OpenShift is via the command line tool _oc_ which is copied
            To login as administrator:
                oc login -u system:admin
 
-    Note that, the IP seen above is dynamic and can change. It can be retrieved with `minishift ip`. Also,
-    instead of 'kvm', you will see 'xhyve' on Mac OS and 'hyperv' on Windows.
+  **Note**:
 
-    **Note:** By default Minishift uses the driver most relevant to the host OS.
-To use a driver of choice for Minishift, use the `--vm-driver=xxx` flag with `minishift start`. For example, to use VirtualBox instead of KVM for Fedora, use `minishift start --vm-driver=virtualbox`.
+  - The IP is dynamically generated for each OpenShift cluster. To check the IP, run the `minishift ip` command.
+  - By default Minishift uses the driver most relevant to the host OS. To use a different driver, set the `--vm-driver` flag in `minishift start`. For example, to use VirtualBox instead of KVM on Linux operating systems, run `minishift start --vm-driver=virtualbox`. For more information about `minishift start` options,
+  see the [minishift start command reference](/docs/minishift_start.md).
 
-1. Add `oc` binary to the _PATH_:
-
-     **Note:** How to modify the _PATH_ varies depending on host OS, version of the OC binary and
-     other variables. In case of doubt, you can check the content of the
-     `~/.minishift/cache/oc` directory.
+1. Add the `oc` binary to the _PATH_ environment variable.
 
         $ export PATH=$PATH:~/.minishift/cache/oc/v1.3.1
 
-1. Login to your OpenShift account and authenticate yourself:
-
-        $ oc login https://192.168.99.128:8443 -u developer -p developer
+  **Note:** Depending on the operating system and the `oc` version, you might need
```

Comment:
> when you do minishift start, in the output, it states that you are logged in as developer, with password developer.

where does it do that? I have not seen that

> I tested with the help of instructions from @praveenkumar, using oc status after I did Minishift start, without loging in again and verified that we did not need to login again.

That's because you where logged in before and your _oc_ session had not expired yet. If you start from scratch also deleting/edting your ~/.kube/config  or letting the access token expire, you will need to login. Login is very much required. There is an issue for improving this and letting minishift log you in - https://github.com/minishift/minishift/issues/349, but right now there is no such thing.

> So we all agreed that this step could be done away with.

I must have missed this discussion

> Let us know if you have different thoughts about this.

See above.